api/__init__.py

```diff
@@ -0,0 +1,54 @@
+import base64
+
+from flask import Flask, request
+import numpy as np
+
+
+def create_app():
+    app = Flask(__name__)
+
+    @app.route('/')
+    def routes():
+        return {
+            "spectrum": "/spectrum",
+            "mean": "/mean",
+            "median": "/median",
+        }
+
+    @app.route('/spectrum', methods=['POST', 'GET'])
+    def spectrum():
+        if request.method == 'GET':
+            return {'params': ['sampling_rate', 'units', 'dtype']}
+
+        req = request.get_json()
+        sr = float(req['sampling_rate'])
+        units = req["units"]
+        dtype = req['dtype'] if 'dtype' in req else "float64"
+        spectrum = np.frombuffer(base64.b64decode(req["spectrum"].encode('utf-8')), dtype=dtype).tolist()
```

Comment:
What if ````spectrum```` is not a proper time series array or is in some way malformed.  Same questions from above related to exception handling and further possible use cases:

- [ ] If this is a time series are missing samples expected and if so how are they handled?

Reply:
Same question for:

- [ ] https://github.com/cwc-products/python-sample/pull/2/files#diff-387e0a038c4ca5f5cc74c8f060afc0a9R39
- [ ] https://github.com/cwc-products/python-sample/pull/2/files#diff-387e0a038c4ca5f5cc74c8f060afc0a9R50

Reply:
#6